core/src/main/res/values-ar/strings.xml

```diff
@@ -361,7 +361,7 @@
   <string name="storage_pref">تخزين</string>
   <string name="storage_sum">الحذف التلقائي والاستيراد والتصدير للحلقة</string>
   <string name="project_pref">المشروع</string>
-  <string name="synchronization_pref">مزامنة</string>
+  <string name="synchronization">مزامنة</string>
```

Comment:
I renamed this because the _pref does not make any sense. The word synchronization is being translated with this key. 
But if there are technical dependencies here then I'll revert it

Reply:
I think the point was that you only need to rename in English. Any changes there will be taken by Transifex and applied to the localised files. So any changes you make to the localised files will be 'overwritten' by Transifex.